.github/workflows/pull_request_tests.yml

```diff
@@ -0,0 +1,18 @@
+name: Unit tests
+on:
+  push:
+    branches:
+      - 'master'
+  pull_request:
+
+jobs:
+  test-python:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: "3.9"  # matches current Python in production
+      - name: "Run Python tests (on Docker)"
+        run: |
+            make clean test-image
```

Comment:
Sorry to ask: should this be `make build-ci` and `make test-ci` ?

Reply:
`make test-ci` triggers the `make build-ci`